specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/examples/ServerSecurityAlertsListByServer.json

```diff
@@ -0,0 +1,30 @@
+{
+    "parameters": {
+        "subscriptionId": "00000000-1111-2222-3333-444444444444",
+        "resourceGroupName": "securityalert-4799",
+        "serverName": "securityalert-6440",
+        "api-version": "2017-03-01-preview"
+    },
+    "responses": {
+        "200": {
+            "body": {
+				"value": [
+                    {
+						"id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/securityalert-4799/providers/Microsoft.Sql/servers/securityalert-6440/securityAlertPolicies",
+						"name": "Default",
+						"type": "Microsoft.Sql/servers/securityAlertPolicies",
+						"properties": {
```

Comment:
add location, kind

Reply:
Added only location for server.
"kind" is not relevant.

Reply:
See above. Adding location here makes the example not align to the definition. please remove.